### ISSUE TITLE
Missing changelog

### ISSUE THREAD

**Issue by robertsLando** [cryptlex/lexactivator-js#43]
Where do I find the changelog of new versions?

**Comment by adnan-kamili** [comment]
This library is just a wrapper around our C library, so there is no business logic here. You can access the changelog of our C library here:

https://docs.cryptlex.com/changelog/lexactivator

**Comment by robertsLando** [comment]
Yeah I know that but there could be also some node/js related changes that are not strictly related to lexactivator C lib

**Comment by adnan-kamili** [comment]
I agree, but they are so infrequent and we don't make any breaking changes, so currently we don't maintain one. 

@azan-n can you please setup auto-changelog creation for this repo

@ahmad-kemsan please ensure that all the commit messages follow use conventional commit format.
